composer.json

```diff
@@ -40,9 +40,11 @@
     "require-dev": {
         "doctrine/coding-standard": "^8.1",
         "jetbrains/phpstorm-stubs": "^2019.1",
-        "phpstan/phpstan": "^0.12.40",
+        "phpstan/phpstan": "^0.12.54",
+        "phpstan/phpstan-phpunit": "^0.12.16",
+        "phpstan/phpstan-strict-rules": "^0.12.5",
         "phpunit/phpunit": "^9.4",
-        "psalm/plugin-phpunit": "^0.10.0",
+        "psalm/plugin-phpunit": "^0.12.0",
         "symfony/console": "^2.0.5|^3.0|^4.0|^5.0",
         "vimeo/psalm": "^3.17.2"
```

Comment:
This PR claims to update Psalm but it doesn't.

Reply:
It does, in the `composer.lock`. But then https://github.com/doctrine/dbal/blob/faf8ddd7e09e495d890a7579f842e5b6fc24aa4a/.github/workflows/static-analysis.yml#L62 should probably be changed too.

Reply:
It should bump the constraint too.